README.md

```diff
@@ -62,214 +61,73 @@ Group Project Navigator is always displayed and provides quick access to most co
-from simple HTML blocks to more sophisticated Project Team and Submission blocks. Also, by using Resource blocks, stage 
-can contribute various resources to the project: grading rubrics, submission templates, or any other relevant documents
-and video.
+Students are graded by `Peer Grading` stage. This stage contains questions, some of these questions can be descriptive, 
+other should contain numerical evaluation of group submissions. Descriptive feedback can be displayed to students in
+`Grade Display` stage, while numerical feedback can be used to calculate grades.
  
-Available stages and their intended use:
+Workgroup is graded as a whole --- each student gets the same mark. 
  
-* `Text` - simple stage showing some author-defined text.
-* `Completion` - almost the same as `Text`, except it requires student to explicitly mark this stage as completed. Use
-     for simple non-graded tasks, i.e. "Get acquainted with the team", "Read related chapters in course textbook", etc. 
-* `Submission` - at this stage students are asked to upload their deliverables. Most of actual group project work are
-   likely to happen during this stage. Use to set up a list of deliverables for an activity and a deadline to upload 
-   them. If deliverable templates are provided to students, the best place to place them is stages of this type.
-* `Team Evaluation` - allows students to provide anonymous feedback to teammates. Intended use is to allow students
-    in the team to provide and receive feedback about their team work, role in the team, communication skills, etc.
-* `Peer Grading` - this stage, in addition to `Submission` stage forms a backbone of group project. During this stage
-    deliverables uploaded earlier are graded by either students in other groups or Teaching Assistants. Grading criteria
-    can be delivered to students as contents of this stage or resources on this stage.
-* `Evaluation Display` - this stage allows students to review feedback from their teammates. Use with `Team Evaluation` stage.
-* `Grade Display` - this stage allows students to review their group grade and feedback received from graders. Use with 
-    `Peer Grading` stage.
-
-Detailed list of available XBlocks is [available in the docs folder](/docs/XBlocks.md).
-
-Group Project XBlock v2 uses standard Studio editing capabilities:
-
-<img src="/docs/images/studio_edit1.png" width="400" alt="Studio screenshot displaying project navigator XBlock, with outlined "View" (red outline) and "Edit" (blue outline) buttons">
-
-* Edit button (blue outline) is used to edit XBlock settings, i.e. "Display name", "Open Date", "Close Date", etc.
-* View button (red outline) allows to "jump into" an XBlock and view it's nested XBlocks, allowing to add, configure, 
-    reorder and delete them.
+Grading can be done either by course staff, or peer-grading schema can be used, in submissions made by workgroup 
+are graded by configurable number of their peers. 
```

Comment:
"Grading can be done either by course staff (staff-grading) or other students (peer-grading)."